src/OpenLoco/Windows/IndustryList.cpp

```diff
@@ -578,7 +578,7 @@ namespace OpenLoco::Ui::Windows::IndustryList
     namespace NewIndustries
     {
 
-        static const Ui::Size window_size = { 578, 172 };
+        static const Ui::Size windowSize = { 578, 172 };
```

Comment:
Same here. Should probably just check all the const items

Reply:
I did this refactor and there were thousands of renames and LoC changed, and this PR is already too big. I'll do it in a separate PR just for windowSize?

Reply:
Kk sounds good